src/twisted/conch/openssh_compat/factory.py

```diff
@@ -66,8 +67,14 @@ def getPrivateKeys(self):
                     privateKeys[key.sshType()] = key
         return privateKeys
 
-    def getPrimes(self):
+    def getPrimes(
+        self,
+    ) -> Optional[Dict[int, List[Tuple[int, int]]]]:  # type:ignore[override]
         try:
             return primes.parseModuliFile(self.moduliRoot + "/moduli")
         except OSError:
+            # As seen in the type:ignore[override] above, this historically
+            # handled errors by (invalidly) returning None, and we should
+            # probably fix this at some point, since callers don't need to
+            # handle None given the contract of the superclass.
```

Comment:
Should we have a ticket for this?

Reply:
I think I'm going to say "no" because it's not entirely clear to me what the fix should be, and I don't want to file a ticket that just gestures vaguely at it. In context the comment just explains the history.